docs/src/install.md

```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
 ```julia
 pkg> add Bloqade
 ```
+## Using Bloqade Optimized AMIs from AWS Marketplace
+The Bloqade team has implemented 2 dedicated AMIs that can be acquired using AWS Marketplace
+
+- Bloqade AMI with Julia
+- Bloqade Optimized Deep Learning AMI with CUDA and Julia
+
+
+### Bloqade AMI with Julia (base image)
+The Bloqade AMI contains:
+- The latest Julia installation using juliaup '1.5.1' which Julia '1.5.1'  for more information please refer to [Juliaup](https://github.com/JuliaLang/juliaup) 
+- The latest version of Bloqade 
+
+### Bloqade Optimized Deep Learning AMI with CUDA and Julia
+In addition to the content of our base image, this image further contains
+- NVIDIA CUDA, cuDNN, NCCL, GPU Drivers, Intel MKL-DNN, Docker, NVIDIA-Docker and EFA support
+- Block devices
```

Comment:
I'm wondering if there is a better way to present this? Maybe put them as a table or at least put the paths into a code block?

Reply:
i was thinking a table but was worried overkill? hmm